.travis/update-types.ts

```diff
@@ -0,0 +1,316 @@
+import fs = require('fs');
+import path = require('path');
+import prettier = require('prettier');
+import got = require('got');
+
+interface RawType {
+  type: string;
+  name: string;
+  description: string;
+}
+
+interface RawComments {
+  events: {
+    [k: string]: {
+      description: string;
+      name: string;
+      returns?: RawType[];
+    }[];
+  };
+  requests: {
+    [k: string]: {
+      description: string;
+      name: string;
+      params?: RawType[];
+      returns?: RawType[];
+    }[];
+  };
+}
+
+type AnyType = PrimitiveType | ObjectType | ArrayType;
+
+interface PrimitiveType {
+  type: 'string' | 'number' | 'boolean';
+  optional: boolean;
+}
+
+interface ObjectType {
+  type: 'object';
+  properties: {
+    [k: string]: AnyType;
+  };
+  optional: boolean;
+}
+
+interface ArrayType {
+  type: 'array';
+  items: PrimitiveType | ObjectType | undefined;
+  optional: boolean;
+}
+
+const DOTS_REGEX = /\./g;
+const outFile = path.join(__dirname, '../types/index.d.ts');
+async function getLatestComments() {
+  const latestReleaseResponse = await got('https://api.github.com/repos/Palakis/obs-websocket/releases/latest', {json: true});
```

Comment:
Yeah, I had mentioned on discord not targeting master.
This seems like a solid solution.